code/script.js

```diff
@@ -1,9 +1,185 @@
+    accordionHeaderTwo.classList.remove("active")
+    accordionHeaderThree.classList.remove("active")
+    accordionHeaderFour.classList.remove("active")
+    accordionHeaderFive.classList.remove("active")
+    accordionHeaderSix.classList.remove("active")
+    accordionHeaderSeven.classList.remove("active")
+});
+
+
+
+
+
+
+accordionHeaderTwo.addEventListener('click', () => {
+    accordionContentOne.classList.remove("accordion-selected")
+    accordionContentTwo.classList.toggle("accordion-selected")
+    accordionContentThree.classList.remove("accordion-selected")
+    accordionContentFour.classList.remove("accordion-selected")
+    accordionContentFive.classList.remove("accordion-selected")
+    accordionContentSix.classList.remove("accordion-selected")
+    accordionContentSeven.classList.remove("accordion-selected")
+});
+
+/* This is the code for the minus sign after clicking on the Question */
+accordionHeaderTwo.addEventListener('click', () => {
+    accordionHeaderOne.classList.remove("active")
+    accordionHeaderTwo.classList.toggle("active")
+    accordionHeaderThree.classList.remove("active")
+    accordionHeaderFour.classList.remove("active")
+    accordionHeaderFive.classList.remove("active")
+    accordionHeaderSix.classList.remove("active")
+    accordionHeaderSeven.classList.remove("active")
+});
+
+
+
+
+accordionHeaderThree.addEventListener('click', () => {
+    accordionContentOne.classList.remove("accordion-selected")
+    accordionContentTwo.classList.remove("accordion-selected")
+    accordionContentThree.classList.toggle("accordion-selected")
+    accordionContentFour.classList.remove("accordion-selected")
+    accordionContentFive.classList.remove("accordion-selected")
+    accordionContentSix.classList.remove("accordion-selected")
+    accordionContentSeven.classList.remove("accordion-selected")
+});
+
+/* This is the code for the minus sign after clicking on the Question */
+accordionHeaderThree.addEventListener('click', () => {
+    accordionHeaderOne.classList.remove("active")
+    accordionHeaderTwo.classList.remove("active")
+    accordionHeaderThree.classList.toggle("active")
+    accordionHeaderFour.classList.remove("active")
+    accordionHeaderFive.classList.remove("active")
+    accordionHeaderSix.classList.remove("active")
+    accordionHeaderSeven.classList.remove("active")
+});
+
+
+
+accordionHeaderFour.addEventListener('click', () => {
+    accordionContentOne.classList.remove("accordion-selected")
+    accordionContentTwo.classList.remove("accordion-selected")
+    accordionContentThree.classList.remove("accordion-selected")
+    accordionContentFour.classList.toggle("accordion-selected")
+    accordionContentFive.classList.remove("accordion-selected")
+    accordionContentSix.classList.remove("accordion-selected")
+    accordionContentSeven.classList.remove("accordion-selected")
+});
+
+/* This is the code for the minus sign after clicking on the Question */
```

Comment:
Maybe this can be included in your functions above instead, since the eventListeners are the same?